sonar-project.properties

```diff
@@ -4,9 +4,9 @@ sonar.projectVersion=0.0.1
 
 sonar.sources= src/app/, src/cases/
 sonar.test.inclusions=**/*.spec.ts
-sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, src/app/shared/services/mc-launch-darkly-service.ts, src/app/shared/services/rootInjectorGuard.ts
+sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, **/*.module.ts
 
 client.sonar.ts.tslint.configPath=tslint.json
 client.sonar.ts.coverage.lcovReportPath=reports/tests/coverage/ng/lcov.info
-sonar.javascript.lcov.reportPaths=reports/tests/coverage/ng/lcov.info
-sonar.host.url=
+sonar.typescript.lcov.reportPaths=reports/tests/coverage/ng/lcov.info
```

Comment:
Leave it as javascript please

Reply:
I didn't make these changes I picked them out from `feature/release2-wa2-a`. Seems like you made the changes.